atomic_reactor/tasks/plugin_based.py

```diff
@@ -31,7 +31,6 @@ class PluginBasedTask(Task[ParamsT]):
     #   {"name": "add_filesystem", "args": {...}}
     # Refer to plugins.json schema for the details.
     plugins_conf: ClassVar[List[Dict[str, Any]]] = []
-    task_name = 'default'
```

Comment:
this is used in : 

        logger.info(r"task %s finished successfully \o/", self.task_name)

Reply:
it is moved to it's parent class, so it is still there